### PR TITLE
MRC-1795: Add endpoint for retrieving calibration options

### PR DESCRIPTION
**Pull request by r-ash**

### CODE REVIEW

File: docker/build
Comment (richfitz) [inline docker/build:32]: Reminder to reset this

**Review by EmmaLRussell — COMMENTED** [review]
Looks good to me. The calibration options won't depend on any parameters, like what input files were chosen?

**Review by richfitz — CHANGES_REQUESTED** [review]
LGTM apart from the branch update once naomi is merged

**Review by richfitz — APPROVED** [review]

**Pull request merged**
